README.md

```diff
@@ -23,22 +23,24 @@ Scavenger provides more sophisticated and clear UI and elaborate the instrumenta
 * Analyze runtime dead code with no code changes.
 * Support JVM based languages.
     * Agent for Java 1.7 is officially not supported any more.
-        * However, if you can not avoid, please
-          use [old agent](https://repo1.maven.org/maven2/com/navercorp/scavenger/scavenger-old-agent-java/1.0.0/scavenger-old-agent-java-1.0.0.jar)
+        * However, if you can not avoid, please use old agent.
```

Comment:
How about using 'cannot' instead of 'can not'?

Reply:
edit in 59cfad1da951be2ad344ec81b027efc964de0b2f